src/bluesearch/server/search_server.py

```diff
@@ -110,32 +110,30 @@ def __init__(
 
         self.logger.info("Initialization done.")
 
-    def _get_model(self, model_name):
+    def _get_model(self, model_name: str) -> EmbeddingModel:
         """Construct an embedding model from its name.
 
         Parameters
         ----------
-        model_name : str
+        model_name :
             The name of the model.
 
         Returns
         -------
-        bluesearch.embedding_models.EmbeddingModel
-            The embedding model class.
+        sentence_embedding_model :
```

Comment:
Should it be `sentence_embedding_model : EmbeddingModel` like in `embedding_models.py`?

Reply:
Yes, nice catch!

Reply:
Done in 5203164